clients/src/main/java/org/apache/kafka/common/requests/OffsetCommitRequest.java

```diff
@@ -144,7 +144,6 @@
 
     // default values for old versions,
     // will be removed after these versions are deprecated
-    @Deprecated
```

Comment:
Removed the `@Deprecated` annotation since both the Kafka broker and the Kafka client need to support old version of the RPC.

Reply:
"// will be removed after these versions are deprecated" seems wrong and we should probably remove it.